mypy.ini

```
@@ -0,0 +1,20 @@
+[mypy]
+cache_dir = .mypy_cache
+allow_redefinition = True
+warn_unused_configs = True
+warn_redundant_casts = True
+show_error_codes = True
+show_column_numbers = True
+check_untyped_defs = True
+follow_imports = silent
+disallow_untyped_calls = True
+disallow_untyped_defs = True
+strict_optional = True
+strict_equality = True
+disallow_subclassing_any = True
+warn_unreachable = True
+no_implicit_optional = True
+disallow_incomplete_defs = True
+strict_concatenate = True
+python_version = 3.10
+plugins = numpy.typing.mypy_plugin
```

リポジトリを確認しましたが，下記以外はpytorchでは使われていないみたいですね．
一旦は下記で良いと思います．このまま反映コミット作成下さい．
```suggestion
[mypy]
python_version = 3.10
cache_dir = .mypy_cache

plugins = numpy.typing.mypy_plugin

allow_redefinition = True
warn_unused_configs = True
warn_redundant_casts = True
show_error_codes = True
show_column_numbers = True
check_untyped_defs = True
follow_imports = normal
local_partial_types = True
enable_error_code = possibly-undefined
warn_unused_ignores = False

strict_optional = True
```